### PR TITLE
marathon-swift 3.0.0

### PR DESCRIPTION
**Pull request by orta**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by orta** [comment]
Ah, hrm, so this requires Xcode 9.3 which is basically high-sierra only I think at this point

File: Formula/marathon-swift.rb [hunk 1]
```diff
@@ -1,28 +1,15 @@
 class MarathonSwift < Formula
   desc "Makes it easy to write, run and manage your Swift scripts"
   homepage "https://github.com/JohnSundell/Marathon"
-  url "https://github.com/JohnSundell/Marathon/archive/1.0.1.tar.gz"
-  sha256 "ebf455159497ae7747784dd9a95b9678dab27db49e8e10c7c6fc2878edcdbce9"
+  url "https://github.com/JohnSundell/Marathon/archive/3.0.0.tar.gz"
+  sha256 "5dd1f495a8d69d9c3450735b771f816a838a040a45c682b0c42ac17105058a6e"
 
-  bottle do
```
Comment (commitay) [inline Formula/marathon-swift.rb]: Restore the bottle block, it should not be edited manually.

**Comment by commitay** [comment]
```
07:17:44 ==> brew test marathon-swift --verbose
07:18:04 ==> FAILED
07:18:04 Testing marathon-swift
07:18:04 /usr/bin/sandbox-exec -f /tmp/homebrew20180421-5381-lcv6dn.sb /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby -W0 -I /usr/local/Homebrew/Library/Homebrew -- /usr/local/Homebrew/Library/Homebrew/test.rb /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/marathon-swift.rb --verbose
07:18:04 ==> /usr/local/Cellar/marathon-swift/3.0.0/bin/marathon create helloWorld import Foundation; print("Hello World") --no-xcode --no-open
07:18:04 🐣  Created script at helloWorld.swift
07:18:04 ==> /usr/local/Cellar/marathon-swift/3.0.0/bin/marathon run helloWorld
07:18:04 💥  Failed to update packages
07:18:04 👉  Make sure you have write permissions to the folder '/tmp/marathon-swift-test-20180421-5382-d5rfrv/.marathon/Packages/'
07:18:04 Error: marathon-swift: failed
07:18:04 Failed executing: /usr/local/Cellar/marathon-swift/3.0.0/bin/marathon run helloWorld
```

File: Formula/marathon-swift.rb [hunk 2]
```diff
@@ -1,28 +1,15 @@
...
-             "--disable-sandbox"
-    else
-      ENV.delete("CC") # https://bugs.swift.org/browse/SR-3151
-      system "swift", "package", "update"
-      system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib"
-    end
-
+    system "swift", "package", "--disable-sandbox", "update"
+    system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib",
+            "--disable-sandbox"
```
Comment (ilovezfs) [inline Formula/marathon-swift.rb]: quote should be aligned under the quote

**Pull request closed without merging**